connector_prestashop_catalog_manager/models/product_template/exporter.py

```diff
@@ -268,6 +267,12 @@ def list_price(self, record):
         else:
             return {"price": str(price_to_export)}
 
+    @changed_by("standard_price")
+    @mapping
+    def cost_price(self, record):
+        wholesale_price = float(f"{record.standard_price:.2f}")
```

Comment:
@AurelijaNorvaisaite as we got info that Presta supports up to 6 decimals, could be worth providing all 6 or number of decimals based on the cost price decimals in the Odoo settings, but still not more than 6.